src/components/SiteHeader/index.js

```diff
@@ -195,7 +203,7 @@ const MenuItems = ({
 };
 
 export const SiteHeader = props => {
```

Comment:
destructure parameter props ?

Reply:
Could do, but I'm passing all props to underlying child components as well and then it's easier to to desctructure, otherwise I find myself having to list all props that I need to pass for each component 🤷‍♂